dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -3202,15 +3224,15 @@
         }
       }
     },
-    "NumberRangeMetadata": {
+    "NumericRangeMetadata": {
```

Comment:
This is potentially SDK-breaking if it already GA'd.

Reply:
Metadata is not GA'd it is something we introduced this preview version.